pkg/registry/core/node/storage/storage.go

```diff
@@ -104,21 +108,7 @@ func NewStorage(optsGetter generic.RESTOptionsGetter, kubeletClientConfig client
 
 	// Build a NodeGetter that looks up nodes using the REST handler
 	nodeGetter := client.NodeGetterFunc(func(ctx context.Context, nodeName string, options metav1.GetOptions) (*v1.Node, error) {
```

Comment:
using a client may make this entire wrapper interface irrelevant.

Reply:
good point 👍🏻